### PR TITLE
added the view functionality

### PR DESCRIPTION
**Pull request by sivaramakrishnan-techconative**

### CODE REVIEW

File: dev-helpers/dev-helper-initializer.js [hunk 1]
```diff
@@ -1,11 +1,36 @@
 /* eslint-disable no-undef */
-window.onload = function() {
+window.onload = function () {
   window["RestImportBundle"] = window["rest-import-ui-bundle"]
   window["RestImportStandalonePreset"] = window["rest-import-ui-standalone-preset"]
   // Build a system
   const ui = RestImportBundle({
     dom_id: "#rest-import-ui",
-    language:'zh',
+    language: 'zh',
```
Comment (vivek-dhayalan) [inline dev-helpers/dev-helper-initializer.js:8]: @Ashokkumar1798 let the language be EN by default

File: examples/angular-app/src/app/app.component.ts [hunk 2]
```diff
@@ -15,7 +15,31 @@ export class AppComponent {
 
     const reactUI = RestImport({
       dom_id: '#rest-import-ui',
-     language:"en"
+     language:"fr",
```
Comment (vivek-dhayalan) [inline examples/angular-app/src/app/app.component.ts:18]: Same applies here.

File: examples/angular-app/angular.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
             ],
             "styles": [
               "src/styles.css",
-              "src/assets/plugins/rest-import-bundle.css"
+              "src/assets/plugins/rest-import-ui.css"
```
Comment (vivek-dhayalan) [inline examples/angular-app/angular.json:29]: have we changed this in file in babel configs as well?

**Review by vivek-dhayalan — COMMENTED** [review]
I still see some of the jsx file are part of the code can you please take care of that as well?

File: dev-helpers/dev-helper-initializer.js (hunk 1 above)
Comment (Ashokkumar1798) [reply dev-helpers/dev-helper-initializer.js]: Changed language to 'en'

File: examples/angular-app/src/app/app.component.ts (hunk 2 above)
Comment (Ashokkumar1798) [reply examples/angular-app/src/app/app.component.ts]: Changed language to 'en'

File: examples/angular-app/angular.json (hunk 3 above)
Comment (Ashokkumar1798) [reply examples/angular-app/angular.json]: Yes.. Now we have the file name as 'rest-import-bundle' in all places

**Review by vivek-dhayalan — APPROVED** [review]

**Pull request merged**